packages/api/tests/models/test-rules-model.js

```diff
@@ -358,6 +362,23 @@ test.serial('Creating a kinesis rule where an event source mapping already exist
   }
 });
 
+test.serial('creating an invalid kinesis type rule does not add event mappings', async (t) => {
+  const { kinesisRule } = t.context;
+
+  const newKinesisRule = cloneDeep(kinesisRule);
```

Comment:
Nit: you shouldn't need to `cloneDeep` because `t.context.kinesisRule` is reset in `beforeEach` for every test

Reply:
Oh yeah that is a good point. Habit I guess!